src/app.py

```diff
@@ -241,6 +250,19 @@ def search(self):
 
         return response
 
+    # Defaults to using the consortium index. Exposes Elasticsearch's `_mapping` endpoint to return details
+    # regarding a specific index.
+    def mapping(self):
+        return self._get_index_mappings('entities')
```

Comment:
Use the configured value from `self.INDICES['default_index']` instead of the hardcoded one.